### ISSUE TITLE
Suggestion: Add an option so that in detailed view, only overworld coords or nether coords are shown

### ISSUE THREAD

**Issue by DanielHe212** [Ninjabrain1/Ninjabrain-Bot#92]
I don't think there's any point to having both displayed at once, and it just takes up more space.

It would be sufficient for most players to only show nether coords or overworld coords based on whether the last f3+c was in nether or overworld.

The only time you really want both displayed at once is right after initial measurement, for judging whether to travel through overworld or double travel through nether, but a blind close enough to warrant overworld travel is rare unless you are blinding without enough obsidian for second portal, in which case it really wouldn't be a time loss to f3+c while running/pearling towards the coords in overworld to show overworld coords, so it would be viable to default to nether coords (even though initial measurement is done in overworld). Alternatively, have an option to show both until, the first non-measurement f3+c.

Also, a keybind to toggle between showing (4,4) and chunk coords would be cool.

**Comment by Ninjabrain1** [comment]
I do not think this is that important, the bot does not take up much space and I do not see much of a reason to make it smaller. Are you on single monitor or what makes this important to you?
